1.8/administration/virtual-networks/isolation.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+nav_title: Isolation
+post_title: Configuring Isolation in Virtual Networks
+menu_order: 20
+---
+
+# iptables rules
+
+DC/OS uses [iptables](http://linux.die.net/man/8/iptables), a fairly high speed, built-in mechanism for filtering traffic in Linux systems, to set up virtual network isolation. We recommend configuring filtering by deploying a homogenous set of rules to all nodes in your infrastructure. In order to simplify this, we also recommend using the [ipset](http://ipset.netfilter.org/ipset.man.html) feature of iptables. 
+
+Set up your own chain that bumps over from the `FORWARD` chain. You can do this by running the following command:
+
+    $ iptables -N dcos-isolation
+
+We want to set up a default deny or a default accept policy between filtered overlays. To set up default deny, run the following:
+
+    $ iptables -A dcos-isolation -j REJECT
+
+To set up default accept, run the following:
+
+    $ iptables -A dcos-isolation -j RETURN
+
+We recommend using the `REJECT` directive as opposed to the `DROP` directive as it makes troubleshooting easier. The default is to allow all.
+
+Use ipset to get onto that chain. Create a `hash:net` type ipset named `overlays` that has all of the overlay networks that you want to restrict traffic from, or to. Then insert the rule:
+
+    $ iptables -I FORWARD -m set --match-set overlays src -m set --match-set overlays dst -j dcos-isolation
+
+This rule says that if a given packet is from any of the overlays and is destined to any other overlay, send it to the `dcos-isolation` rule. It can be advantageous to avoid filtering traffic from an overlay to itself. In order to avoid this, we recommend adding an exception set of type `hash:net,net` and adding entries for networks that should not be filtered. Modify the rule to: 
```

Comment:
I agree that providing a reason would be good here. @asridharan or @mhausenblas , can you explain why an administrator would want to avoid doing this?